e2e/button/Button.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Button } from '@gpn-prototypes/vega-button';
+
+describe('Button', () => {
+  describe('primary view', () => {
```

Comment:
я так понимаю, это пропс и его значение
может быть в таком формате будем писать?

```suggestion
  describe('view="primary"', () => {
```